feat: initial implementation for discriminator (#5012)
SAP/cloud-sdk-js: .changeset/gorgeous-toys-kick.md

---
'@sap-cloud-sdk/openapi-generator': minor
---

> [New Feature] Add support for discriminator properties in"one of" and "all of" schemas. Also add support for incorrect usage of the discriminator property with schemas of type "object".

Author: We don't have a matching rule for this, but the rule for code font does not apply here I think (https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-code-font).

I will to rephrase it to make it apply better ;) 
```suggestion
[New Feature] Add support for discriminator properties in schemas that include `oneOf` and `anyOf`. Also add support for incorrect usage of the discriminator property with schemas of type `object`.
```

> [New Feature] Add support for discriminator properties in schemas that include `oneOf` and `anyOf`. Also add support for incorrect usage of the discriminator property with schemas of type `object`.

Author: ```suggestion
[New Functionality] Add support for discriminator properties in schemas that include `oneOf` and `anyOf`. Also add support for incorrect usage of the discriminator property with schemas of type `object`.
```
